packages/design-system/src/components/pro-badge.tsx

```diff
@@ -0,0 +1,26 @@
+import { theme } from "../stitches.config";
+import { Text } from "./text";
+
+export const ProBadge = () => {
```

Comment:
Didn't you want to reuse it without putting it into design system?

Reply:
You wanted to put it into design system 🤷

Reply:
I was talking about regarding a generic badge/chip component, but idk we do have some not generic component there too, but I guess we don't have to.